util/linuxfw/linuxfw_unsupported.go

```diff
@@ -9,9 +9,15 @@
 package linuxfw
 
 import (
+	"errors"
+
 	"tailscale.com/types/logger"
 )
 
+// ErrUnsupported is the error returned from all functions on non-Linux
+// platforms.
+var ErrUnsupported = errors.New("unsupported")
```

Comment:
linuxfw:unsupported, just in case we actually end up logging this somewhere, it'll be much easier to indentify where it came from.